spec/site_prism/element_spec.rb

```diff
@@ -16,6 +16,12 @@
     it { is_expected.to respond_to(:wait_until_element_one_visible) }
     it { is_expected.to respond_to(:wait_until_element_one_invisible) }
 
+    it 'supports negated rspec existence matchers' do
```

Comment:
Can we re-write these using the element stubs and turn them into 1liners, and add both options i.e.
```
it 'supports rspec existence matchers'
  expect(subject).to have_element_two # I think
end
```

Reply:
I would like to keep the one expectation that the negated matcher is called. Would that be ok?

```ruby
it 'supports rspec existence matchers' do                                
  expect(subject).to have_element_one                                    
end                                                                      
                                                                         
it 'supports negated rspec existence matchers' do                        
  expect(subject).to receive(:has_no_element_two?).once.and_call_original
  expect(subject).not_to have_element_two                                
end                                                                      
```

Reply:
Added in a74be32